yaml_specs/modulemd_packager_v2.yaml

```diff
@@ -3,15 +3,18 @@
 # for a module stream. It is always a proper subset of a `document: modulemd`
 # of the same `version`.
 document: modulemd-packager
```

Comment:
One more comment... Once the `modulemd_stream_v2.yaml` text is settled, it would be best to tediously replicate the applicable pieces here in `modulemd_packager_v2.yaml` as well.

Reply:
Yeah, it probably would. :sob: